### ISSUE TITLE
Case Study: File Drag to Desktop

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#190]
```
David from box.net will provide a case study for drag to desktop saving.

```

Original issue reported on code.google.com by `sethladd@google.com` on 9 Sep 2010 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
See if I will be in the watchlist by adding comments here.
```

Original comment by `davidtwt...@gmail.com` on 9 Sep 2010 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
Seth, I just shared the Google Doc with you at your google.com email address. 
Can you access it?
```

Original comment by `davidtwt...@gmail.com` on 15 Sep 2010 at 7:10

**Comment by GoogleCodeExporter** [comment]
```
Just saw it, thanks!
```

Original comment by `sethladd@google.com` on 15 Sep 2010 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
Any comments? Things you like me to modify to be consistent with the style of 
other articles? Any technical things you want me to add?
```

Original comment by `davidtwt...@gmail.com` on 16 Sep 2010 at 7:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `v...@google.com` on 24 Sep 2010 at 5:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 26 Sep 2010 at 12:50
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Patch to add the converted article.
```

Original comment by `han...@google.com` on 27 Sep 2010 at 9:19

Attachments:
- [html5rocks_rev307.patch](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-190/comment-7/html5rocks_rev307.patch)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 4321389859.
```

Original comment by `ericbide...@html5rocks.com` on 27 Sep 2010 at 8:44
- Changed state: **Fixed**
